prover/Makefile

```diff
@@ -44,7 +44,11 @@ corset:
 ## Build and bundle the Corset trace-expander dependency
 ##
 zkevm/arithmetization/zkevm.bin: corset
-	cd ../constraints && $(CORSET) make zkevm.bin && mv zkevm.bin ../prover/zkevm/arithmetization
+	cd ../constraints && \
+	BRANCHES=$$(git branch -r --contains HEAD | sed 's|origin/rel/||') && \
```

Comment:
IIUC, BRANCHES contains all the branches containing the target commit. This includes the "right" branch but will also include all the future branches. 

E.g. if we run the same comment in 5 months it will return all versions >= `v0.8.0-rc8.1`

Reply:
Switched to tags now :)